scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1744,14 +1751,22 @@ function Set-ProvisioningMode {
     Update-ConfigYaml({
         param($configurationYaml)
 
+        if ($DynamicReprovisioning) {
+            $DynamicReprovisioning = 'true'
+        }
+        else {
+            $DynamicReprovisioning = 'false'
+        }
+
         if ($ManualConnectionString -or $ManualX509) {
-            $selectionRegex = '(?:[^\S\n]*#[^\S\n]*)?provisioning:\s*#?\s*source:\s*".*"\s*#?\s*device_connection_string:\s*".*"'
+            $selectionRegex = '(?:[^\S\n]*#[^\S\n]*)?provisioning:\s*#?\s*source:\s*".*"\s*#?\s*device_connection_string:\s*".*"\s*#?\s*dynamic_reprovisioning:\s*".*"'
```

Comment:
did ManualX509 support dynamic reprovisioning?

Reply:
Users have the option of enabling dynamic reprovisioning with the manual modes too. It's just that the daemon will stop execution if a reprovisioning scenario occurs and users will have to manually update the config file again.

This isn't really useful in any way but it was decided to just allow the config in all modes as it doesn't really make a difference